$NetBSD: patch-sapi_fpm_www.conf.in,v 1.1 2025/02/08 02:57:59 taca Exp $

* Use UNIX domain socket default and create with default group.
* Set proper default configuration values with PHP version.

--- sapi/fpm/www.conf.in.orig	2019-01-09 09:54:13.000000000 +0000
+++ sapi/fpm/www.conf.in
@@ -33,7 +33,8 @@ group = @php_fpm_group@
 ;                            (IPv6 and IPv4-mapped) on a specific port;
 ;   '/path/to/unix/socket' - to listen on a unix socket.
 ; Note: This value is mandatory.
-listen = 127.0.0.1:9000
+;listen = 127.0.0.1:9000
+listen = @php_fpm_localstatedir@/run/php-fpm@PHP_VER@
 
 ; Set listen(2) backlog.
 ; Default Value: 511 (-1 on FreeBSD and OpenBSD)
@@ -45,8 +46,9 @@ listen = 127.0.0.1:9000
 ; Default Values: user and group are set as the running user
 ;                 mode is set to 0660
 ;listen.owner = @php_fpm_user@
-;listen.group = @php_fpm_group@
-;listen.mode = 0660
+listen.group = @php_fpm_group@
+listen.mode = 0660
+
 ; When POSIX Access Control Lists are supported you can set them using
 ; these options, value is a comma separated list of user/group names.
 ; When set, listen.owner and listen.group are ignored
@@ -250,7 +252,7 @@ pm.max_spare_servers = 3
 
 ; The access log file
 ; Default: not set
-;access.log = log/$pool.access.log
+;access.log = @php_fpm_localstatedir@/log/php-fpm@PHP_VER@.$pool.access.log
 
 ; The access log format.
 ; The following syntax is allowed
@@ -314,7 +316,7 @@ pm.max_spare_servers = 3
 ; The log file for slow requests
 ; Default Value: not set
 ; Note: slowlog is mandatory if request_slowlog_timeout is set
-;slowlog = log/$pool.log.slow
+;slowlog = @php_fpm_localstatedir@/log/php-fpm@PHP_VER@.$pool.log.slow
 
 ; The timeout for serving a single request after which a PHP backtrace will be
 ; dumped to the 'slowlog' file. A value of '0s' means 'off'.
